action.yaml

```
@@ -13,5 +13,6 @@ inputs:
     default: 'false'
 runs:
   using: docker
-  image: docker://ghcr.io/akuity/kargo-render:v0.1.0-rc.37
-  args: action
+  image: docker://ghcr.io/akuity/kargo-render:v0.1.0-rc.39
+  args:
+  - action
```

```suggestion
  image: docker://ghcr.io/akuity/kargo-render:v0.1.0-rc.39
  args: action
```

Will take the bump, but I am more hesitant about the other change since I find it unlikely that Actions has had a breaking change of that magnitude. We can revisit that and fix it separately when there's more clarity on it.